xml/article_nfs_storage.xml

```diff
@@ -450,255 +454,231 @@ include /etc/drbd.d/*.res;</screen>
-  op monitor interval="30s"
-&prompt.crm.conf;<command>clone</command> cl-exportfs_root exportfs_root</screen>
-      <remark>toms 2016-09-01: What clientspec should be used here?</remark>
-      <para>
-       This resource does not hold any actual NFS-exported data, merely the
-       empty directory (<filename>/srv/nfs</filename>) that the other NFS
-       exports are mounted into. Since there is no shared data involved
-       here, you can safely <emphasis>clone</emphasis> this resource.
-      </para>
-     </listitem>
-     <listitem>
-      <para>
-       Since any data should be exported only on nodes where this clone has
-       been properly started, add the following constraints to the
-       configuration:
-      </para>
-<screen>&prompt.crm.conf;<command>order</command> o-root_before_nfs Mandatory: \
-  cl-exportfs_root g-nfs:start
-&prompt.crm.conf;<command>colocation</command> c-nfs_on_root inf: \
-  g-nfs cl-exportfs_root
-&prompt.crm.conf;<command>commit</command></screen>
-      <para>
-       After this, Pacemaker should start the NFSv4 virtual file system root
-       on both nodes.
-      </para>
-     </listitem>
-     <listitem>
-      <para>
-       Check the output of the <command>exportfs -v</command> command to
-       verify this:
-      </para>
-      <screen>&prompt.root;<command>exportfs</command> -v
-/srv/nfs/work   <replaceable>IP_ADDRESS_OF_CLIENT</replaceable>(...)
-/srv/nfs        <replaceable>IP_ADDRESS_OF_CLIENT</replaceable>(...)</screen>
-     </listitem>
-    </orderedlist>
-   </sect3>
--->
-
-<!--   <sect3 xml:id="sec-ha-quick-nfs-resources-nfsexport-nonroot">
-    <title>NFS exports</title>-->
-    <para>
-     To export the <filename>/srv/nfs/work</filename> directory to clients,
-     use the following primitive:
-    </para>
-    <orderedlist>
-     <listitem>
-      <para>
-       Create NFS exports with the following commands:
-      </para>
-<screen>&prompt.crm.conf;<command>primitive exportfs_work \
-  ocf:heartbeat:exportfs \
-    params directory="/srv/nfs/work" \
-      options="rw,mountpoint" \
-      clientspec="&nfs-clientspec;" \
-      wait_for_leasetime_on_stop=true \
-      fsid=100 \
-  op monitor interval="30s"</command></screen>
-     </listitem>
-     <listitem>
-      <para>
-       After you have created these resources, append them to the existing
-       <literal>g-nfs</literal> resource group:
-      </para>
-<screen>&prompt.crm.conf;<command>modgroup g-nfs add exportfs_work</command></screen>
-     </listitem>
-     <listitem>
-      <para>
-       Commit this configuration:
-      </para>
-<screen>&prompt.crm.conf;<command>commit</command></screen>
+   <procedure>
+    <title>Creating supporting resources for NFS</title>
+    <step>
+     <para>
+      Create a primitive to manage the NFS server daemon:
+     </para>
+<screen>&prompt.crm.conf;<command>primitive p_nfsserver nfsserver \
+  params nfs_server_scope=SUSE \
```

Comment:
It's my understanding from the discussion in the bug that this line covers point 6 from the TID (consistent host name). I may have misunderstood, though, so let me know if we do still need a note about the `/etc/systemd/system/nfs-server.service.d/cluster.conf` config.

Reply:
You are right. `nfs_server_scope`  is the implementation of RA nfsserver to cover the same purpose of `/etc/systemd/system/nfs-server.service.d/cluster.conf`.

Reply:
Great, thanks for confirming!